CHANGELOG.rst

```diff
@@ -1,7 +1,51 @@
 Version 0.17.0 ()
 ---------------------------
 
```

Comment:
There is a bit of text here which say what has happened and who has contributed, I have pasted an example for 0.16.3 below:

``` rst
This release adds an enhancement and compatibility changes with upstream libraries. Thanks to `@raphaelquast <https://github.com/raphaelquast>`_, `@droumis <https://github.com/droumis>`_ and `@hoxbro <https://github.com/hoxbro>`_.
```